packages/blade/src/components/Typography/Code/__tests__/Code.native.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Code } from '../Code';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+
+describe('<Code />', () => {
+  it('should render Code with default properties', () => {
+    const { getByText, toJSON } = renderWithTheme(<Code>TEST_TOKEN</Code>);
+    expect(getByText('TEST_TOKEN')).toBeTruthy();
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render large Code with different types', () => {
+    const { toJSON } = renderWithTheme(<Code size="small">SMALL</Code>);
+    expect(toJSON()).toMatchSnapshot();
```

Comment:
Can we add an assertion on a property that gets affected by `size="small"`, it would be easier than reviewing snapshot diffs

Reply:
Thanks for this suggestion!! Added `.toHaveStyle` assertions and they are so much better! We might as well remove snapshots and rely on them in future.

Reply:
Nice, maybe instead of hard coding 12 you could use the theme token if that seems better

Reply:
done